app/views/reservations/_form.html.erb

```diff
@@ -9,7 +9,7 @@
       </div>
     </div>
     <%= f.input :number_of_guests %>
-    <% if current_user == Reservation.find(params[:id]).planet.user %>
+    <% if @planet.reservations.any? && @planet.reservations.any? { |reservation| reservation.user == current_user } %>
```

Comment:
I think this should be the following since this will only appear for the planet host (not the reservation.user)... let me know what you think though!
if @planet.reservations.any? && Reservation.find(params[:id]).planet.user %>

Reply:
Yeah, but both me and Amane are getting some errors. This params[:id] refers to the planet id, and not the reservation one. That's why I first checked if the planet has any reservations and if it does, check if any of the reservations belongs to the host. If so, then it should appear.

Reply:
Ah I see!! Then would it be the following?
(My concern is that this status checkbox is only supposed to appear for the planet host, not the reservation user)
<% if @planet.reservations.any? && @planet.user == current_user  %>

Reply:
Oh yeah, this works too, and actually is more readable than what I proposed! 👍